src/pybind/mgr/dashboard/controllers/prometheus.py

```diff
@@ -135,6 +135,14 @@ def list(self, cluster_filter=False, **params):
             return self.alert_proxy('GET', f'/alerts?filter=cluster={fsid}', params)
         return self.alert_proxy('GET', '/alerts', params)
 
+    @RESTController.Collection(method='GET')
+    def url(self):
+        return Settings.PROMETHEUS_API_HOST
+
+    @RESTController.Collection(method='GET', path='/alertmanager_url')
+    def alertmanager_url(self):
+        return Settings.ALERTMANAGER_API_HOST
+
```

Comment:
we don't need this right? there are existing api's that already returns the url if I am not wrong?

Reply:
I thought so, but couldn't find any appropiate @nizamial09

Reply:
i commented below the urls. will that work?

Reply:
Yeah of course... Thanks Nizam